CC/include/Parallel.h

```diff
@@ -15,31 +15,33 @@
 //#                                                                        #
 //##########################################################################
 
-#ifndef PARALLEL_SORT_HEADER
-#define PARALLEL_SORT_HEADER
-
-#ifdef ParallelSort
-#undef ParallelSort
-#pragma message "Replacing preprocessor symbol 'ParallelSort' with the one defined in Parallel.h"
-#endif
+#ifndef CCCORE_PARALLEL_HEADER
+#define CCCORE_PARALLEL_HEADER
+#include <atomic>
```

Comment:
Does this belong here?

Reply:
eagle eyes: I simply forgot to remove that after experiments

Reply:
This is why code reviews rock!